### PR TITLE
v5.29 changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Comment by amyblais** [comment]
@mkraft For v5.29 changelog, please help with:

1. Any new config.json settings
2. Database changes
3. Websocket Event changes
4. API changes

File: source/administration/changelog.md [hunk 1]
```diff
@@ -4,6 +4,87 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+### Open Source Components
+ - Removed ``@types/react-custom-scrollbars`` from https://github.com/mattermost/mattermost-webapp.
+
+### Database Changes
+ - 
+
+### API Changes
+ - 
+
+### Websocket Event Changes
```
Comment (amyblais) [inline source/administration/changelog.md:67]: Waiting for dev to provide these details.

**Comment by amyblais** [comment]
@mkraft Kind reminder to provide these details for the v5.29 changelog https://github.com/mattermost/docs/pull/4017#issuecomment-720477400.

File: source/administration/changelog.md [hunk 2]
```diff
@@ -4,6 +4,90 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+   - Added ``ManagedResourcePaths``, to add support for a setting to use with the Desktop Managed Resources feature.
+   
+### Go Version
+ - 5.29 is built with Go ``1.14.6``.
+
+### Open Source Components
+ - Removed ``@types/react-custom-scrollbars`` from https://github.com/mattermost/mattermost-webapp.
+
+### Database Changes
+ -
```
Comment (esethna) [inline source/administration/changelog.md:65]: There is a new table for threads. Please ask @reflog to provide details here on database changes for threads.

**Comment by mkraft** [comment]
> @mkraft For v5.29 changelog, please help with:
> 
> 1. Any new config.json settings
> 2. Database changes
> 3. Websocket Event changes
> 4. API changes

# Notes from the "new release" notes block in pull requests (added part way through the release)

- Added ability to retrieve compliance files from system console ([#15825](https://github.com/mattermost/mattermost-server/pull/15825), [@sbishel](https://github.com/sbishel))
- Added configuration setting to ignore Guests when using SAML and synchronizing with LDAP. ([#15936](https://github.com/mattermost/mattermost-server/pull/15936), [@sbishel](https://github.com/sbishel))
- Added new API endpoints for working with experimental collapsed threads ([#16091](https://github.com/mattermost/mattermost-server/pull/16091), [@reflog](https://github.com/reflog))
- Added new local API endpoints for getting, updating and deleting incoming and outgoing webhooks ([#16094](https://github.com/mattermost/mattermost-server/pull/16094), [@mgdelacroix](https://github.com/mgdelacroix))
- Added plugin hooks for `ReationHasBeenAdded` and `ReactionHasBeenRemoved` ([#15765](https://github.com/mattermost/mattermost-server/pull/15765), [@shieldsjared](https://github.com/shieldsjared))
- Added the ManagedResourcePaths setting for use with the Desktop Managed Resources feature ([#16213](https://github.com/mattermost/mattermost-server/pull/16213), [@hmhealey](https://github.com/hmhealey))
- All system admins will be sent an email in their preferred locale for the following cases (assuming an unpaid, cloud licensed installation):
  1. The installation is at 9 users, and a user joins (AT limit)
  2. The installation is at 10 users, and a user joins (OVER limit)
  3. The installation is at 9 active users, and an inactive user is activated (AT limit)
  4. The installation is at 10 active users, and an inactive user is activated (OVER limit) ([#16053](https://github.com/mattermost/mattermost-server/pull/16053), [@nickmisasi](https://github.com/nickmisasi))
- Automatically disable channel wide mentions when a user mutes a channel ([#15992](https://github.com/mattermost/mattermost-server/pull/15992), [@sowmiyamuthuraman](https://github.com/sowmiyamuthuraman))
- Build process improvements to the MMCTL download process to use AWS S3 location instead of GitHub Releases. This avoids issues with GitHub rate limiting. ([#15993](https://github.com/mattermost/mattermost-server/pull/15993), [@jaydeland](https://github.com/jaydeland))
- Fixed a bug where a session will hold on to a cached value
  in an HA setup with read-replicas configured. ([#16170](https://github.com/mattermost/mattermost-server/pull/16170), [@agnivade](https://github.com/agnivade))
- Fixed case where duplicate sidebar categories could be created on first use of experimental sidebar ([#16030](https://github.com/mattermost/mattermost-server/pull/16030), [@hmhealey](https://github.com/hmhealey))
- Removed some unnecessary conditions from of the compliance export query to improve the performance. ([#15757](https://github.com/mattermost/mattermost-server/pull/15757), [@mkraft](https://github.com/mkraft))
- Updated `icon_emoji` field in incoming webhooks to allow for emojis to also be specified with surrounding colons ([#15753](https://github.com/mattermost/mattermost-server/pull/15753), [@edtrist](https://github.com/edtrist))

# New config settings

* `ServiceSettings.EnableAWSMetering`
* `ServiceSettings.ThreadAutoFollow`
* `ServiceSettings.ManagedResourcePaths`

# Database

* Altered some types and defaults in `SidebarCategories` table
* Added column `Threads.ChannelId`

# Websocket

No changes.

# API

No changes.

**Review by mkraft — APPROVED** [review]

**Review by thefactremains — APPROVED** [review]

**Pull request merged**